packages/matchbox/src/components/Modal/Legacy.js

```diff
@@ -0,0 +1,178 @@
+import React, { useRef } from 'react';
+import { Transition } from 'react-transition-group';
+import FocusLock from 'react-focus-lock';
+import ScrollLock, { TouchScrollable } from 'react-scrolllock';
+import styled from 'styled-components';
+import { padding, maxWidth } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+import PropTypes from 'prop-types';
+import { tokens } from '@sparkpost/design-tokens';
+import { Close } from '@sparkpost/matchbox-icons';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
+import { WindowEvent } from '../WindowEvent';
+import { Button } from '../Button';
+import { Box } from '../Box';
+import { Portal } from '../Portal';
+import { onKey } from '../../helpers/keyEvents';
+import { secondsToMS } from '../../helpers/string';
+import { isInIframe } from '../../helpers/window';
+import { base, focusLock, wrapper, content, contentAnimation, closeButton } from './styles';
+
+const StyledBase = styled('div')`
+  ${base}
+  ${padding}
+`;
+
+const StyledWrapper = styled('div')`
+  ${wrapper}
+`;
+
+const StyledCloseButton = styled(Button)`
+  ${closeButton}
+`;
+
+const StyledFocusLock = styled(FocusLock)`
+  ${focusLock}
+`;
+
+const StyledContent = styled('div')`
+  ${content}
+  ${contentAnimation}
+`;
+
+const Legacy = React.forwardRef(function Legacy(props, userRef) {
+  const {
+    onClose,
+    children,
+    portalId,
+    className,
+    showCloseButton,
+    maxWidth,
+    open,
+    ...rest
+  } = props;
+  const container = useRef();
+  const content = useRef();
+
+  const handleKeydown = e => {
+    if (open && onClose) {
+      onKey('escape', onClose)(e);
+    }
+  };
+
+  const handleOutsideClick = e => {
+    const isOutside =
+      content &&
+      !content.current.contains(e.target) &&
+      container &&
+      container.current.contains(e.target);
+
+    if (open && isOutside && onClose) {
+      onClose(e);
+    }
+  };
+
+  return (
+    <>
+      <ScrollLock isActive={open} />
+      <Portal containerId={portalId}>
+        <TouchScrollable>
+          <StyledBase
+            open={open}
+            {...rest}
+            className={className}
+            onClose={onClose}
+            role="dialog"
+            aria-modal="true"
+          >
+            {/*
+              Ref can't be a direct child of TouchScrollable
+              See https://github.com/jossmac/react-scrolllock/issues/67
+            */}
+            <Box p={['400', null, '700']} size="100%" ref={container}>
+              <StyledWrapper>
+                <Box
```

Comment:
For some reason the old modal isn't expanding to 100% max width like the old modal, but the changes look right: See https://github.com/SparkPost/matchbox/commit/d419005cd0426f26eef1cebec91eb19b72e5c402

Reply:
![Screen Shot 2020-07-15 at 4 01 08 PM](https://user-images.githubusercontent.com/3903325/87590363-a9003f00-c6b4-11ea-8d42-c725be8821a8.png)